x/posts/internal/keeper/keeper_test.go

```diff
@@ -87,6 +108,7 @@ func TestKeeper_SavePost(t *testing.T) {
 			store := ctx.KVStore(k.StoreKey)
 			for _, p := range test.existingPosts {
 				store.Set([]byte(types.PostStorePrefix+p.PostID.String()), k.Cdc.MustMarshalBinaryBare(p))
+				store.Set([]byte(types.LastPostIDStoreKey), k.Cdc.MustMarshalBinaryBare(p.PostID))
```

Comment:
I'm not sure this is the best thing to do. Probably what we could do is add a new field into the `tests` structure called `LastPostID` that is used here inside the `p.PostID` value. This would allow for better control of the setup.